packages/@lwc/engine/src/shared/format.ts

```diff
@@ -18,7 +18,7 @@ export function getComponentTag(vm: UninitializedVM): string {
     }
 }
 
-export function getComponentStack(vm: VM): string {
+export function getComponentStack(vm: UninitializedVM): string {
```

Comment:
`getComponentStack` and `getErrorComponentStack` are almost identical, the only difference is the formatting. Do you think it's possible to merge the method together to avoid the code duplication? Changing the component stack formatting sounds like a low impact code change.

Reply:
they were unified before, then split for some reasons that I don't remember... 

I think we should keep them separate, it is not in the scope of this PR to try to unify that. open an issue to investigate more.

Reply:
Here is the git issue to track this work: https://github.com/salesforce/lwc/issues/1695